lib/modules/platform/gitlab/index.ts

```diff
@@ -414,6 +418,11 @@ export async function getBranchStatus(
     return BranchStatus.yellow;
   }
   logger.debug(`Got res with ${branchStatuses.length} results`);
+
+  const mrStatus = (await getBranchPr(branchName))?.headPipelineStatus;
+  if (!is.undefined(mrStatus)) {
+    branchStatuses.push({ status: mrStatus as BranchState, name: '' });
```

Comment:
doesn't the head pipeline object also has a name, wich can be used here? if not use a synthetic name like `head_pipeline` or something like that.

Reply:
Does not seems so https://docs.gitlab.com/ee/api/merge_requests.html#get-single-mr